eval/simple_eval.py

```diff
@@ -42,6 +42,9 @@ def f1_emnlp2020(
     if len(predictions) == 0:
         return (1.0, 0.0, 0.0)
 
+    predictions = list(set(predictions))
+    gold = list(set(gold))
```

Comment:
We should not need to dedup the gold, because the gold does not contain duplicates. I verified that removing this line still leads to the same performance.

before (with line 45, 46):

```
                prec    recall  f1
ExactMetric     10.44   6.53    4.24
BLEUMetric      24.44   17.66   16.07
ROUGEMetric     41.11   33.77   32.38

```

before (with line 45 only):

```
                prec    recall  f1
ExactMetric     10.44   6.53    4.24
BLEUMetric      24.44   17.66   16.07
ROUGEMetric     41.11   33.77   32.38

```